archaius-core/src/main/java/com/netflix/archaius/Property.java

```diff
@@ -48,7 +48,7 @@
     /**
      * @return  Most recent value for the property
      */
-    T get(T defaultValue);
+    T get();
```

Comment:
It might be worth also having a getOrElse(T else) method so that you could override the general default for a specific call.